cypress/e2e/1-getting-started/property.cy.js

```diff
@@ -0,0 +1,42 @@
+describe('Check specific properties of the API response', () => {
+  it('Check individual properties', () => {
+    cy.request({
+      method: 'GET',
+      url: 'http://localhost:3001/customers?page=1&limit=10&size=All'
+    }).then((response) => {
+      // Check response status
+      expect(response.status).to.eq(200);
+
+      // Check that the response has 'customers' and 'pageInfo'
+      expect(response.body).to.have.property('customers').that.is.an('array').with.length(10);
```

Comment:
Quando uma _assertion_ é tão longa, recomendo quebrar a linha, para ajudar na legibilidade e evitar _scroll_ lateral

Por exemplo:

```suggestion
      expect(response.body)
        .to.have.property('customers')
        .that.is.an('array')
        .with.length(10);
```

Reply:
Que tal?